docs/_docs/restapi.adoc

```diff
@@ -564,11 +558,6 @@ http://host:port/ignite?cmd=decr&cacheName={cacheName}&key={key}&init={init_valu
 |Description
 |Example
 
-|`cacheName`
```

Comment:
Need to remove the "&cacheName={cacheName}" parameter from the URL example as well.